### DISCUSSION TITLE
What is the biggest change in NextJS version 14 and version 15?

### DISCUSSION THREAD

**Discussion by khuongtai** [vercel/next.js#70306] [Show and tell]
I have a question what is the biggest change between nextjs 14 and 15?

**Comment by romhenri** [comment]
The biggest changes between Next.js 14 and 15 include full support for React 19, which enhances performance, and the introduction of Partial Pre-Rendering (PPR) for combining static and dynamic content on the same page. Additionally, Next.js 15 offers improved middleware, caching strategies, and better image optimization​.
